src/components/inputs/Inputs.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { InputsListContainer } from '../../containers/InputsList';
+import Empty from '../empty/Empty';
```

Comment:
why not add index.ts to empty package, so you would import from '../empty' instead of '../empty/Empty